docs/notebooks/getting_started/particles.ipynb

```diff
@@ -0,0 +1,903 @@
+    "# Using PlasmaPy Particles"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0c7eeced",
+   "metadata": {},
+   "source": [
+    "[plasmapy.particles]: ../../particles/index.rst\n",
+    "\n",
+    "The [plasmapy.particles] subpackage contains functions to access basic particle data and classes to represent particles."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "24125310",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from plasmapy.particles import *"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "650d57e8",
+   "metadata": {},
+   "source": [
+    "## Contents\n",
+    "\n",
+    "1. [Particle properties](#Particle-properties)\n",
+    "2. [Particle objects](#Particle-objects)\n",
+    "3. [Custom particles](#Custom-particles)\n",
+    "4. [Molecules](#Molecules)\n",
+    "5. [Particle lists](#Particle-lists)\n",
+    "6. [Dimensionless particles](#Dimensionless-particles)\n",
+    "7. [Nuclear reactions](#Nuclear-reactions)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e224284a",
+   "metadata": {},
+   "source": [
+    "## Particle properties"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7696deba",
+   "metadata": {},
+   "source": [
+    "There are several functions that provide information about different particles that might be present in a plasma. The input of these functions is a representation of a particle, such as a string for the atomic symbol or the element name."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "006b8d3e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "atomic_number(\"Fe\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2aa5af34",
+   "metadata": {},
+   "source": [
+    "[atomic number]: https://en.wikipedia.org/wiki/Atomic_number\n",
+    "\n",
+    "We can provide a number that represents the [atomic number]."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "69958326",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "element_name(26)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "cca89cef",
+   "metadata": {},
+   "source": [
+    "We can also provide standard symbols or the names of particles."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "16ca5b6f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "is_stable(\"e-\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "35d8e49e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "charge_number(\"proton\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e498e01e",
+   "metadata": {},
+   "source": [
+    "[alpha particle]: https://en.wikipedia.org/wiki/Alpha_particle\n",
+    "\n",
+    "The symbols for many particles can even be used directly, such as for an [alpha particle]. To create an \"α\" in a Jupyter notebook, type `\\alpha` and press tab."
```

Comment:
Is there a list of these special particle names in the docs somewhere? I would create it (if it doesn't exist already) and then link to it here.

Reply:
There isn't yet a table of special particle names, but there's an open issue on it (#1222).